Classes/Command/QueueCommandController.php

```diff
@@ -59,6 +59,23 @@ public function listCommand()
     }
 
     /**
+     * Show details of a queue
+     *
```

Comment:
just some very (stupid) description + option doc, so that the help screen looks nicer ...